templates/corporate/case-studies/tum-case-study.md

```diff
@@ -32,9 +32,10 @@ when Tobias came across Zulip.
 ## “Better user experience than Slack”
 
 Tobias evaluated Zulip by [visiting the Zulip development
-community](/try-zulip/) to see it in action. “It takes a bit of
-time to get used to, but Zulip has the best user experience of all the
-chat apps I’ve tried,” Tobias says. “With the discussion organized by
+community](https://chat.zulip.org/?show_try_zulip_modal)
```

Comment:
@alya does this link want to be here, or to the main development community page?

Reply:
Not sure. Probably https://zulip.com/development-community/ is fine here.

Reply:
Added a small commit (first commit) for this change.